src/components/common/dropdown/MenuDropdown.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuItem,
+} from "@/components/ui/dropdown-menu";
+import { cn } from "@/lib/utils";
+
+interface Option {
+  label: React.ReactNode;
+  value: string;
+  ariaLabel?: string;
+}
+
+/**
+ * MenuDropdown
+ * - 수정/삭제 같은 액션을 위한 메뉴 드롭다운 컴포넌트
+ * - trigger prop으로 원하는 요소를 넘겨서 사용
+ * - 옵션 label에 ReactNode 허용 (이미지, 아이콘 등)
+ */
+interface MenuDropdownProps {
+  options: Option[]; // 표시할 메뉴 항목들
+  onSelect: (value: string) => void; // 항목 클릭 시 실행할 콜백
+  trigger: React.ReactElement;
+}
+
+export default function MenuDropdown({
+  options,
+  onSelect,
+  trigger,
+}: MenuDropdownProps) {
+  return (
+    <DropdownMenu>
+      {/* 드롭다운 트리거 버튼 */}
+      <DropdownMenuTrigger asChild>{trigger}</DropdownMenuTrigger>
+      {/* 드롭다운 메뉴 영역 */}
+      <DropdownMenuContent
+        align="end"
+        className="min-w-[101px] md:w-[126px] rounded-2xl border border-gray-200 bg-white px-1.5 py-1.5 space-y-1"
+      >
+        {options.map((option) => (
+          <DropdownMenuItem
+            aria-label={option.ariaLabel}
```

Comment:
옵셔널에는 언제나 fallback값을 고려하는 습관을 두시는게 좋습니다
ariaLabel은 옵셔널이니까 아래처럼 제안드려요
```suggestion
            aria-label={option.ariaLabel ?? option.value}
```